pkg/mindpak/build/packer.go

```diff
@@ -109,7 +109,7 @@ func (_ *Packer) InitBundle(opts *InitOptions) (*mindpak.Bundle, error) {
 	if err := bundle.Manifest.Write(f); err != nil {
 		return nil, fmt.Errorf("writing manifest data: %w", err)
 	}
-	fmt.Printf("wrote to %s", f.Name())
+	fmt.Printf("wrote to %s\n", f.Name())
```

Comment:
We should probably not be using `Printf` at all. But this was already here, so let's take this fix as-is. If we really need to output something we should take the output stream as a parameter instead.